manim/mobject/mobject.py

```diff
@@ -62,9 +62,6 @@ def __init__(self, **kwargs):
         self.generate_points()
         self.init_colors()
 
-    def __str__(self):
```

Comment:
Question: Why are we removing this? It doesn't seem to help this issue (which is about VGroup and VDict).

Reply:
If by keeping this, you do a `print(grp)`, you'll get the same old output of `VGroup` because when both str and repr are defined and you do a `print(grp)` (as opposed to a `repr(grp)`) it uses the `__str__` method. For VGroup and VDict, the str method is inherited from Mobject and hence we remove it from the Mobject itself.
One might ask, "Why not use repr(grp) for printing the VGroup?" I think print is more common and more people are more likely to use it as first instinct.

Reply:
Ok I see. I'd suggest implementing this as `Mobject.__repr__` instead of deleting it. In this way, all mobjects will still have a string representation, and your other changes will make it so that `VGroup.__repr__` overrides `Mobject.__repr__`.

Reply:
I thought we would be implementing __repr__ for Mobjects in a separate PR, no? Since for a Mobject, there isn't anything specific you can print. Like for a Square, it would be "Square with side length = 2" but for circle it would be "Circle with radius 2". That being said, still feel free to suggest what could possibly go in Mojects's repr.

Reply:
Even if we do take care of that in another PR, this PR shouldn't leave all mobjects without a str/repr method.

Reply:
Just copy the current `__str__` into a `__repr__` for now.

Reply:
Yes, that's a good solution.